sbc_dashboard_usage/sbc_dashboard_usage.py

```diff
@@ -0,0 +1,237 @@
+###################################################################
+# Script Name   : sbc_dashboard_usage.py
+#
+#
+# Description : Microservice script to read table
+#             : data from the looker internal database and
+#             : save it to csv in S3.
+#
+# Requirements: You must set the following environment variables
+#             : to establish credentials for the microservice user
+#             : export lookeruser=<<looker user>>
+#             : export lookerpass=<<looker_PASSWD>>
+#
+# Usage       : python sbc_dashboard_usage.py sbc_dashboard_usage.json
+
+
+import mysql.connector as connection
+from mysql.connector.errors import Error
+import logging
+import lib.logs as log
+from tzlocal import get_localzone
+from pytz import timezone
+import os  # to read environment variables
+import os.path  # file handling
+import sys  # to read command line parameters
+import boto3  # s3 access
+from botocore.exceptions import ClientError
+import json  # to read json config files
+from io import StringIO
+import pandas as pd
+import datetime
+from lib.redshift import RedShift
+
+# Get script start time
+local_tz = get_localzone()
+yvr_tz = timezone('America/Vancouver')
+yvr_dt_start = (yvr_tz
+  .normalize(datetime.datetime.now(local_tz)
+  .astimezone(yvr_tz)))
+    
+logger = logging.getLogger(__name__)
+log.setup()
+logging.getLogger("RedShift").setLevel(logging.WARNING)
+
+
+# Exit and return exit code, message
+def clean_exit(code, message):
+  """Exits with a logger message and code"""
+  logger.info(f'Exiting with code {code} : {message}')
+  sys.exit(code)# Exit and return exit code, message
+def clean_exit(code, message):
```

Comment:
@BJenning Looks like clean_exit is here twice

Reply:
Ive removed the redefinition.